tests/langchain/test_langchain_autolog.py

```diff
@@ -771,6 +771,26 @@ def test_langchain_autolog_callback_injection_in_batch(invoke_arg, config, async
             assert set(handler.logs) == {"chain_start", "chain_end"}
 
 
+def test_tracing_source_run_in_batch():
+    mlflow.langchain.autolog()
```

Comment:
```suggestion
    mlflow.langchain.autolog(log_models=True)
```

You need to pass `log_models=True` otherwise a model won't be logged from autolog.